src/apps/pillarx-app/components/TokensPercentage/TokensPercentage.tsx

```diff
@@ -0,0 +1,27 @@
+// components
+import BodySmall from '../Typography/BodySmall';
+
+// images
+import TriangleGreenIcon from '../../images/triangle-up-green.svg';
+import TriangleRedIcon from '../../images/triangle-down-red.svg';
+
+
+type TokensPercentageProps = {
+    percentage: number;
+}
+
+const TokensPercentage = ({percentage}: TokensPercentageProps) => {
```

Comment:
Can we guarantee percentage gets a value when called? Just looking at this from an isolated component point of view where we may use this elsewhere

Reply:
I've added the potentiality of it being undefined, check new code :)

Reply:
thanks!